### PR TITLE
onboarding sophia 2023

### PR DESCRIPTION
**Pull request by span1012**
### Description
Testing out git for onboarding: created new branch 

Replace this with description

## Test Plan

Replace this with test plan

### CODE REVIEW

File: onboarding2023/sophia_onboarding.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Sophia Nexus Onboarding 2023
```
Comment (Shreya1233) [inline onboarding2023/sophia_onboarding.txt:1]: please make this fix rn

**Review by Shreya1233 — CHANGES_REQUESTED** [review]
plz fix this

File: onboarding2023/sophia_onboarding.txt (hunk 1 above)
Comment (vinnyh125) [inline onboarding2023/sophia_onboarding.txt:1]: This looks good

**Review by vinnyh125 — APPROVED** [review]
Looks awesome!!

**Review by Shreya1233 — APPROVED** [review]

**Pull request merged**